### PR TITLE
test555

### PR DESCRIPTION
**Pull request by Wthongkom**

### CODE REVIEW

File: src/test/java/HelloThere/YOLO.java [hunk 1]
```diff
@@ -6,6 +6,9 @@ public static void main(String[] args) {
 
 		System.out.println("Hello");
 		System.out.println("I fix it 2");
+		
+		
+		System.out.println("test 555");  //keaw
```
Comment (luckloa) [inline src/test/java/HelloThere/YOLO.java:11]: fix it

Comment (Wthongkom) [reply src/test/java/HelloThere/YOLO.java:11]: I will fix it

File: src/test/java/HelloThere/YOLO.java [hunk 2]
```diff
@@ -6,6 +6,9 @@ public static void main(String[] args) {
 
 		System.out.println("Hello");
```
Comment (luckloa) [inline src/test/java/HelloThere/YOLO.java:7]: Change Hello to whatever

**Comment by luckloa** [comment]
Need to fix it

**Pull request closed without merging**